bindings/python/tools/python/ostk/astrodynamics/utilities.py

```
@@ -19,7 +22,144 @@
+    dr_y: float
+    dr_z: float
+    dv: float
+    dv_x: float
+    dv_y: float
+    dv_z: float
+
+
+def compute_residuals(
+    candidate_states: list[State],
+    reference_states: list[State],
+    local_orbital_frame_factory_or_frame: LocalOrbitalFrameFactory | Frame = Frame.GCRF(),
```

Ruff (0.14.5)

Do not perform function call `Frame.GCRF` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)